test TestCreateProjectThruWebhookProxyJenkinsFile fails on ODS box

**Describe the bug**
test TestCreateProjectThruWebhookProxyJenkinsFile fails on ODS box

**To Reproduce**
Steps to reproduce the behavior:
1. Run tests on ODS box
2. See error

**Expected behavior**
test should pass

**Affected version (please complete the following information):**
 - OpenShift: 3.11
 - OpenDevStack latest master

**Log Output (ensure to remove any confidential information like tokens, project names, etc.**
```
    amazon-ebs: === RUN   TestCreateProjectThruWebhookProxyJenkinsFile
    amazon-ebs: project unitt-test deleted - project.project.openshift.io "unitt-test" deleted
    amazon-ebs:
    amazon-ebs: project unitt-dev deleted - project.project.openshift.io "unitt-dev" deleted
    amazon-ebs:
    amazon-ebs: project unitt-cd deleted - project.project.openshift.io "unitt-cd" deleted
    amazon-ebs:
    amazon-ebs: --- FAIL: TestCreateProjectThruWebhookProxyJenkinsFile (37.87s)
    amazon-ebs:     jenkinsfile_test.go:40: Unable to remove build configs: exit status 1
    amazon-ebs: FAIL
    amazon-ebs: FAIL	github.com/opendevstack/ods-core/tests/create-projects	125.177s
    amazon-ebs: FAIL
```

**Additional context**
Add any other context about the problem here.

--- a/tests/utils/clean-up.go
+++ b/tests/utils/clean-up.go
@@ -41,6 +41,7 @@ func RemoveBuildConfigs(projectName string, buildConfigName string) error {
 		"-n", projectName,
 		"delete",
 		"bc", buildConfigName,
+		"--ignore-not-found", "true",
 	}, dir, []string{})
 	if err != nil {
 		return err

